targets/TARGET_TOSHIBA/TARGET_TMPM3HQ/spi_api.c

```diff
@@ -0,0 +1,284 @@
+/* mbed Microcontroller Library
```

Comment:
@ganesh-ramachandran : I see the SPI API implemented but don't see the pin definitions in PinNames.h .. any reason??